cmd/podman/buildfarm/list.go

```diff
@@ -0,0 +1,119 @@
+package buildfarm
+
+import (
+	"fmt"
+	"os"
+	"sort"
+
+	"github.com/containers/common/pkg/completion"
+	"github.com/containers/common/pkg/config"
+	"github.com/containers/common/pkg/report"
+	"github.com/containers/podman/v4/cmd/podman/common"
+	"github.com/containers/podman/v4/cmd/podman/registry"
+	"github.com/containers/podman/v4/cmd/podman/validate"
+	"github.com/spf13/cobra"
+)
+
+var (
+	buildfarmLsDescription = `podman buildfarm ls
+
+List all available farms. The output of the farms can be filtered
```

Comment:
You switch between buildfarms and farms, I think you should just stick to buildfarms.

Reply:
I think we should stay with farms to keep it generic in the event we realise we want to use farms for more than just builds in the future. I will go through and switch everything from buildfarms to farms where I have used both by mistake.